### ISSUE TITLE
Can't deactivate plugin

### ISSUE THREAD

**Issue by richardsawyer** [anthony-mills/s3-video#12]
When I try to deactivate it, I get an error Access denied for user ''@'localhost' (using password: NO)
Screenshot: http://screencast.com/t/4Nn97N7lKc
Nothing else on the page.
Any ideas?

**Comment by anthony-mills** [comment]
Hi Richard, given the error I assume the plugin is having trouble connecting with the database server to remove the tables it created. You can just delete the plugin manually and leave the tables it will not have any adverse affect on your wordpress install. If you would like to remove all traces of the plugin though, simply delete the plugin code base and then use PHPMyAdmin or similar to delete the following tables from your Wordpress database s3_video_playlists, s3_video_playlist_videos, s3_video_analytics, s3_video_stills

Cheers

Anthony

**Comment by richardsawyer** [comment]
Hi Anthony, Actually, your plugin is working perfectly and I like it. I'm having trouble with another plugin so tried to deactivate s3-video as part of troubleshooting that but got stuck. If I delete the plugin manually then reinstall, will it just pick up the tables and resume working?

**Comment by anthony-mills** [comment]
Yeah don't worry about it if you want it back. Just add it removing the tables is just housekeeping to keep in line with Wordpress best practice should you want it gone forever.

**Comment by paulywill** [comment]
@anthony-mills I am also receiving some errors as well when trying to deactivate the plugin:


```
Warning: mysql_query(): Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2) in /home/content/63/7508063/html/wp-content/plugins/s3-video/includes/plugin_setup.php on line 36

Warning: mysql_query(): A link to the server could not be established in /home/content/63/7508063/html/wp-content/plugins/s3-video/includes/plugin_setup.php on line 36
Can't connect to local MySQL server through socket '/var/lib/mysql/mysql.sock' (2)
```

I may have changed my IAM settings on AWS, would this be the cause or is it a local issue with the table?

You mentioned to Richard above to move the plugin manually, is that one folder with the plugins folder?

**Comment by anthony-mills** [comment]
Hi Paul,
            In your case I doubt it is related to IAB change by the look of the error it looks like its trying to connect to a local MySQL socket but failing Perhaps the MySQL server is only configured to accept connections over TCP?

But yes the plugin can be removed manually by deleting the plugin folder directly over ftp or similar from the main plugins folder of your site i.e /wp-content/plugins.
